src/renderers/__tests__/ReactEmptyComponent-test.js

```diff
@@ -173,16 +152,8 @@ describe('ReactEmptyComponent', () => {
         }
       }
 
-      var instance1 =
-        <TogglingComponent
-          firstComponent={'div'}
-          secondComponent={Child}
-        />;
-      var instance2 =
-        <TogglingComponent
-          firstComponent={Child}
-          secondComponent={'div'}
-        />;
+      var instance1 = <TogglingComponent firstComponent={'div'} secondComponent={Child} />;
```

Comment:
Btw, the way prettier works is to cram as much as possible within the limit. So it's usually not a good idea to go too far away from 80 columns as the result tend to look crowded.